app/views/reports/index.html.erb

```diff
@@ -4,23 +4,19 @@
 
 <div id="content">
   <div>
-    <% if can_create_new_report(@project) %>
+    <% if can_manage_reports?(@project) %>
       <%= link_to new_project_reports_path(@project), class: 'btn btn-primary', id: 'new-report-btn', 'data-no-turbolink' => true do %>
         <span class="glyphicon glyphicon-plus" aria-hidden="true"></span>
         <span class="hidden-xs"><%=t "projects.reports.index.new" %></span>
       <% end %>
-
       <%= link_to "", remote: true, class: "btn btn-default", id: "edit-report-btn" do %>
         <span class="glyphicon glyphicon-pencil" aria-hidden="true"></span>
         <span class="hidden-xs"><%=t "projects.reports.index.edit" %></span>
       <% end %>
-      <% if can_delete_reports(@project) %>
-        <%= link_to "", remote: true, class: "btn btn-default", id: "delete-reports-btn" do %>
-          <span class="glyphicon glyphicon-remove" aria-hidden="true"></span>
-          <span class="hidden-xs"><%=t "projects.reports.index.delete" %></span>
-        <% end %>
+      <%= link_to "", remote: true, class: "btn btn-default", id: "delete-reports-btn" do %>
```

Comment:
Probably, we still want to check if user can manage reports here right? Otherwise anybody with read access to this page could delete reports, or at least see the button.

Reply:
There's already a parent check `can_manage_report?` present.

Reply:
Aah, :banana:. Then okay, no fix needed.